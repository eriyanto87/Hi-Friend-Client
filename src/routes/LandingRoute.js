import React from "react";
import Landing from "../components/Landing";

export default function LandingRoute() {
  return <Landing />;
}
