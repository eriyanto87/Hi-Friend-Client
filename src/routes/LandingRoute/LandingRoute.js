import React from "react";
import Landing from "../../components/Landing/Landing";

export default function LandingRoute() {
  return <Landing />;
}
