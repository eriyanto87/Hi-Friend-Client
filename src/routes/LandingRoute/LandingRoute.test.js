import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import LandingRoute from "./LandingRoute";

describe("Landing Route", () => {
  it("Landing Snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <LandingRoute />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <LandingRoute />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
