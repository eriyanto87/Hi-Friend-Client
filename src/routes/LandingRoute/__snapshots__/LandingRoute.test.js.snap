// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Route Landing Snapshot 1`] = `
Array [
  <div
    className="landing"
  >
    <section
      className="messenger"
    >
      <header
        className="messenger-header"
      >
        <h1
          className="messenger-title"
        >
          Hi, Friend!
        </h1>
      </header>
    </section>
    <main
      className="chat"
    >
      <section
        className="left-message-bubble"
      >
        <div
          className="bot-name"
        >
          BOT
        </div>
        <p
          className="message"
        >
          Hi, Friend! Welcome to my app. I am a chatbot! Please sign in using these sign in credentials.
          <p>
            Username: Demo 
            <br />
            Password: Password123!
            <br />
            <a
              href="/signin"
              onClick={[Function]}
            >
              <button>
                 Sign in 
              </button>
            </a>
          </p>
        </p>
      </section>
      <section
        className="left-message-bubble"
      >
        <div
          className="bot-name"
        >
          BOT
        </div>
        <p
          className="message"
        >
          You can also click on the Register button below to sign up for an account if you choose not to sign in using our demo account!
          <a
            href="/registration"
            onClick={[Function]}
          >
            <button>
               Register 
            </button>
          </a>
        </p>
      </section>
    </main>
    <form
      className="input-area"
    >
      <input
        className="input"
        placeholder="Type your message here..."
        type="text"
      />
      <button
        className="send"
      >
        SEND
      </button>
    </form>
  </div>,
  <div
    className="footer"
  >
    Â© 2020
  </div>,
]
`;
